tests/test_workflow_graphgen_scoring.py

```
@@ -0,0 +1,127 @@
+    assert agg["result"] == "insufficient data"
+
+
+def test_aggregate_axis_excludes_exploratory_records():
+    """Exploratory records must not contaminate range computation or winner declaration.
+
+    Cursor Bugbot #3365141204 — aggregate_axis includes every record without
+    checking the exploratory flag, so a cheap exploratory A+B run would make A+B
+    look better than it is and block the real winner from being credited.
+    """
+    records = []
+    # 5 fair A records: tokens_in 100–104 (cheaper)
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)